src/WebJobs.Extensions.CosmosDBCassandra/Config/CosmosDBCassandraWebJobsBuilderExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.CosmosDBCassandra;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Extension methods for CosmosDB integration.
+    /// </summary>
+    public static class CosmosDBCassandraWebJobsBuilderExtensions
+    {
+        /// <summary>
+        /// Adds the CosmosDB extension to the provided <see cref="IWebJobsBuilder"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IWebJobsBuilder"/> to configure.</param>
+        public static IWebJobsBuilder AddCosmosDBCassandra(this IWebJobsBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            builder.AddExtension<CosmosDBCassandraExtensionConfigProvider>()
+                .ConfigureOptions<CosmosDBCassandraOptions>((config, path, options) =>
+                {
+                    // TODO: Add to Options the Cassandra connection details
+                    options.ConnectionString = config.GetConnectionString(Constants.DefaultConnectionStringName);
```

Comment:
This is here to provide default settings. This allows a user to set an environment variable (name defined by `Constants.DefaultConnectionStringName`) that if the user sets it, he does not need to set it on each Function. If we want to support that, then make sure `Constants.DefaultConnectionStringName` points to something that makes sense for Cassandra (maybe the contact point?) or if you need more things (Contact point + usr+ pwd), then add these as different items in `CosmosDBCassandraOptions` with different constant names, and read them here.